src/components/LoginOptions/LoginOptions.stories.mdx

```diff
@@ -0,0 +1,178 @@
+  component={LoginOptions}
+/>
+
+# Login Options
+
+The `LoginOptions` component displays the options that are available for a user to authenticate when starting a form.
+
+## Presentation
+
+The `LoginOptionsDisplay` component shows the different buttons that can be clicked to start the authentication process.
+Next to each button, the corresponding logo is displayed.
+
+export const DisplayTemplate = ({loginAsYourselfOptions, loginAsGemachtigdeOptions}) => (
+  <LoginOptionsDisplay
+    loginAsYourselfOptions={loginAsYourselfOptions}
+    loginAsGemachtigdeOptions={loginAsGemachtigdeOptions}
+  />
+);
+
+<Canvas>
+  <Story
+    name="Display"
+    args={{
+      loginAsYourselfOptions: [
+        {
+          identifier: "digid",
+          label: "DigiD",
+          logo: {
+              href: "https://www.digid.nl/",
+              title: "DigiD",
+              imageSrc: "/digid.png"
+          },
+          url: "http://localhost:8999/auth/form-slug/digid/start?next=http%3A%2F%2Flocalhost%3A3000%2F%3F_start%3D1",
+        },
+        {
+          identifier: "eHerkenning",
+          label: "eHerkenning",
+          logo: {
+              href: "https://www.eherkenning.nl/",
+              title: "eHerkenning",
+              imageSrc: "/eherkenning.png"
+          },
+          url: "http://localhost:8999/auth/form-slug/eherkenning/start?next=http%3A%2F%2Flocalhost%3A3000%2F%3F_start%3D1",
+        },
+        {
+          identifier: "eidas",
+          label: "eIDAS",
+          logo: {
+              href: "https://www.eherkenning.nl/",
+              title: "eIDAS",
+              imageSrc: "/eidas.png"
+          },
+          url: "http://localhost:8999/auth/form-slug/eidas/start?next=http%3A%2F%2Flocalhost%3A3000%2F%3F_start%3D1",
+        },
+        {
+          identifier: "anonymous",
+          label: "Begin form",
+          url: "#",
+        }
+      ],
+      loginAsGemachtigdeOptions: [{
+	      identifier: "digid_machtigen",
+	      label: "DigiD Machtigen",
+        logo: {
+            href: "https://www.digid.nl/",
+            title: "DigiD Machtigen",
+            imageSrc: "/digid.png"
+        },
+        url: "http://localhost:8999/auth/issue-2447-style-login-page/digid_machtigen/start?next=http%3A%2F%2Flocalhost%3A3000%2F%3F_start%3D1",
+      }]
+    }}
+  >
+    {DisplayTemplate.bind({})}
+  </Story>
+</Canvas>
+
+### Props
+
+<ArgsTable of={LoginOptionsDisplay} />
+
+## Functional
+
+The `LoginOptions` component handles the 'smart' part of the component. It separates the login options between
+the ones that can be used to 'login as yourself' and the ones that can be used to log in 'on behalf of someone else'.
+It also formats the labels that will be displayed on the buttons and creates the right URL to which the user will be
+redirected upon clicking one of the buttons.
+
+export const Template = ({form, onFormStart}) => (
+  <LiteralsProvider literals={{'beginText': {resolved: 'Begin Form'}}}>
```

Comment:
Probably would be a bit neater to use the decorator approach here for the provider, see for example:

* https://github.com/open-formulieren/open-forms-sdk/blob/main/src/components/LanguageSelection/LanguageSelection.stories.mdx?plain=1#L160
* https://github.com/open-formulieren/open-forms-sdk/blob/main/src/components/LanguageSelection/LanguageSelection.stories.mdx?plain=1#L166